Database/data.js

```diff
@@ -0,0 +1,249 @@
+    views: faker.datatype.number(),
+    date: faker.date.past(),
+    subscribers: faker.datatype.number(),
+    likes: faker.datatype.number(),
+    dislikes: faker.datatype.number(),
+    description: faker.commerce.productDescription()
+  },
+  {
+    id: "KgOYSy-q3m0",
+    title: "Asus Zenfone 6 - ULTIMATE Rotating Camera Smartphone.",
+    author: "Mrwhosetheboss",
+    avatar:
+      "https://yt3.ggpht.com/ytc/AAUvwnjkCEa45O-45oyqizcbgxfZwbL6OeZ_CIZCPQsE6Q=s68-c-k-c0x00ffffff-no-rj",
+    thumbnail:
+      "https://i.ytimg.com/vi/KgOYSy-q3m0/hqdefault.jpg?sqp=-oaymwEcCPYBEIoBSFXyq4qpAw4IARUAAIhCGAFwAcABBg==&rs=AOn4CLB7Cz3T6-4XJ11QllhxCBIokfsf7Q",
+    views: faker.datatype.number(),
+    date: faker.date.past(),
+    subscribers: faker.datatype.number(),
+    likes: faker.datatype.number(),
+    dislikes: faker.datatype.number(),
+    description: faker.commerce.productDescription()
+  },
+    {
+    id: "WVVRVtJ9ujk",
+    title: "20 Crazy Smartphone Gadgets for 2020.",
+    author: "Mrwhosetheboss",
+    avatar:
+      "https://yt3.ggpht.com/ytc/AAUvwnjkCEa45O-45oyqizcbgxfZwbL6OeZ_CIZCPQsE6Q=s68-c-k-c0x00ffffff-no-rj",
+    thumbnail:
+      "https://i.ytimg.com/vi/WVVRVtJ9ujk/hqdefault.jpg?sqp=-oaymwEcCPYBEIoBSFXyq4qpAw4IARUAAIhCGAFwAcABBg==&rs=AOn4CLAbHMqzUq2Z2555sCv5wxrS0fND4A",
+    views: faker.datatype.number(),
+    date: faker.date.past(),
+    subscribers: faker.datatype.number(),
+    likes: faker.datatype.number(),
+    dislikes: faker.datatype.number(),
+    description: faker.commerce.productDescription()
+  },
+    {
+    id: "w_OcYyQmPBk",
+    title: "The $100,000 Smartphone Unboxing.",
+    author: "Mrwhosetheboss",
+    avatar:
+      "https://yt3.ggpht.com/ytc/AAUvwnjkCEa45O-45oyqizcbgxfZwbL6OeZ_CIZCPQsE6Q=s68-c-k-c0x00ffffff-no-rj",
+    thumbnail:
+      "https://i.ytimg.com/vi/w_OcYyQmPBk/hqdefault.jpg?sqp=-oaymwEcCPYBEIoBSFXyq4qpAw4IARUAAIhCGAFwAcABBg==&rs=AOn4CLDV3beG-bmj4_I21Vj80yOuNjKzxA",
+    views: faker.datatype.number(),
+    date: faker.date.past(),
+    subscribers: faker.datatype.number(),
+    likes: faker.datatype.number(),
+    dislikes: faker.datatype.number(),
+    description: faker.commerce.productDescription()
+  },
+    {
+    id: "hsOv_VU3wKA",
+    title: "Xiaomi Mi Mix Alpha - Day in the Life!",
+    author: "Mrwhosetheboss",
+    avatar:
+      "https://yt3.ggpht.com/ytc/AAUvwnjkCEa45O-45oyqizcbgxfZwbL6OeZ_CIZCPQsE6Q=s68-c-k-c0x00ffffff-no-rj",
+    thumbnail:
+      "https://i.ytimg.com/vi/hsOv_VU3wKA/hqdefault.jpg?sqp=-oaymwEcCPYBEIoBSFXyq4qpAw4IARUAAIhCGAFwAcABBg==&rs=AOn4CLCqiQLz9bGPmQ0PrCnrHDKJI12cag",
+    views: faker.datatype.number(),
+    date: faker.date.past(),
+    subscribers: faker.datatype.number(),
+    likes: faker.datatype.number(),
+    dislikes: faker.datatype.number(),
+    description: faker.commerce.productDescription()
+  }
+]
```

Comment:
Hi Mohammad,
Here instead of hardcoding the details for the data you are getting from the faker, 
you could have made an array of data that has the necessary information, looped it, and then filled in the data with the faker. So that for the data generated by the faker you wouldn't have to write it manually for each and every object.

Reply:
I have fixed this issue.